Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
@DomT4 Delicately I'd like to suggest that the world's biggest corporation probably doesn't need taught a lesson in delivering software security from people with our level of experience. I'd also like to suggest the reason that they haven't done so is because the security vulnerabilities are far more speculative than actual.